docs/topics/proc_getting-the-launcher-tool-url-and-credentials.adoc

```diff
@@ -8,7 +8,7 @@
 [id='getting-the-launcher-tool-url-and-credentials_{context}']
 = Getting the {name-launcher} tool URL and credentials
 
-You need the {name-launcher} tool URL and user credentials to create and deploy boosters on {OpenShiftLocal}. This information is provided when the {OpenShiftLocal} is started.
+You need the {name-launcher} tool URL and user credentials to create and deploy examples on {OpenShiftLocal}. This information is provided when the {OpenShiftLocal} is started.
```

Comment:
@huntley: Not sure if you have changed it already, the word "example applications" sounds better here since we talk about deploying it on OpenShiftLocal.